packages/closer/locales/tdf/en.json

```
@@ -623,6 +1319,50 @@
+  "governance_voting_requirements": "Voting Requirements",
+  "governance_connect_wallet_to_vote": "Connect your wallet to vote",
+  "governance_must_be_member": "You must be a member to vote on proposals",
+  "governance_content_label": "Content",
+  "governance_status_draft": "DRAFT",
+  "governance_status_active": "ACTIVE",
+  "governance_status_passed": "PASSED",
+  "governance_status_failed": "FAILED",
+  "governance_status_unknown": "UNKNOWN",
+  "governance_voting_ended": "Voting ended",
+  "governance_days_hours_remaining": "{days} days, {hours} hours remaining",
+  "governance_hours_remaining": "{hours} hours remaining",
```

Biome (2.1.2)

[error] The key governance_content_label was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)